src/containers/customChartsDemos/StackedBarChartDemo/StackedBarChartDemo.tsx

```diff
@@ -2,30 +2,35 @@ import React from 'react';
 import { useMappedState } from 'redux-react-hook';
 import StackedBarChart from '../../../components/customCharts/StackedBarChart';
 import SimpleHorizontalBarChart from '../../../components/customCharts/SimpleHorizontalBarChart';
+import { State } from "../../../types";
 
-const mapState = (state: any): any => ({
-  blokchain: state.tezos.blocks
+const mapState = (state: State): any => ({
+  tezos: state.tezos.blocks,
+  ethereum: state.ethereum.blocks,
 });
 
-function SimpleBarChartDemo(): React.ReactElement {
-  const { blokchain } = useMappedState(mapState);
+const totalWalletsAmounts = (groupingKey: string, blockchain: any) =>
+  blockchain.reduce((acc: any, next: any): any => {
+    const alreadyExist = acc.find(
+      (item: any) => item.key === next[groupingKey]
+    );
+    if (alreadyExist) {
+      alreadyExist.value += next.amount;
+      return acc;
+    }
+    const newItem = {
+      key: next[groupingKey],
+      value: next.amount,
+    };
+
+    return [...acc, newItem];
+  }, []);
```

Comment:
Bardzo kosztowne operacje, ja bym to wrzucil w reducera, zeby nie liczyc tego za kazdym razem jak user wyjdzie i wroci z powrotem na route'a.

Reply:
Dobry pomysł chociaż ten komponent jest tylko po to żeby pokazać jak działa sam wykres (dlatego też nie używałem Materiala i teksty wyglądają jak wyglądają). Nie jestem pewien w jakim kierunku to teraz dalej pójdzie (jak będzie faktycznie ten wykres użyty) więc nie chciałem zmieniać nic w logice poza nim.